tests/modules/malt/run/main.nf

```diff
@@ -8,18 +8,19 @@ include { MALT_RUN   } from '../../../../modules/malt/run/main.nf'
 
 workflow test_malt_run {
 
-    fastas = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
-    gff = file(params.test_data['sarscov2']['genome']['genome_gff3'], checkIfExists: true)
-    seq_type = "DNA"
-    map_db = [ [], file("https://software-ab.informatik.uni-tuebingen.de/download/megan6/megan-nucl-Jan2021.db.zip", checkIfExists: true) ]
+    fastas        = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+    seq_type      = "DNA"
+    map_accession = [ [], file("https://software-ab.informatik.uni-tuebingen.de/download/megan6/nucl_acc2tax-Jul2019.abin.zip", checkIfExists: true) ]
```

Comment:
can this go to nf-core/testdatasets?

Reply:
It's freaking huge, so I prefer to leave it from 'the source' as this _shouldn't_ disappear